.travis.yml

```diff
@@ -70,7 +70,7 @@ install:
 
   # TODO: Do we still need the beta version or can we do simply -c openeye openeye-toolkits?
   # Use beta version for partial bond orders
-  # TODO: Reactivate conda install instead of pip install once OpenEye is released for python 3.7 on conda-forge.
+  # TODO: Reactivate conda install instead of pip install once OpenEye is released for python 3.7 on conda-forge, but I don't think we need oemmtools anymore.
```

Comment:
Yeah, it's good to keep these notes here. This will be useful when we rebuild our testing system up according to the latest cookiecutter in the near future (#237)